### PR TITLE
Finishing up ICoredebug module4 and process11

### PR DESCRIPTION
**Pull request by mikelle-rogers**
## Summary

ICorDebugModule4 and ICorDebugProcess11 are not supported on .NET Framework.

Continuing this [pull request] (https://github.com/dotnet/docs/pull/29732)

### CODE REVIEW

File: docs/framework/unmanaged-api/debugging/icordebugprocess11-interface.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+[Supported in .NET Framework 4.5.2 and later versions.]  
+  
+ Provides a method that configures the debugger to handle in-memory metadata updates in the target process.  
+  
+## Methods  
+  
+|Method|Description|  
+|------------|-----------------|  
+|[EnumerateLoaderHeapMemoryRegions Method](icordebugprocess11-enumerateloaderheapmemoryregions-method.md)|Sets a value that determines how the debugger handles in-memory updates to metadata within the target process.|
```
Comment (noahfalk) [inline docs/framework/unmanaged-api/debugging/icordebugprocess11-interface.md]: This text still refers to the old incorrect description

File: docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ topic_type:
 ---
 # ICorDebugProcess11::EnumerateLoaderHeapMemoryRegions Method
 
-[Supported in .NET Framework 4.5.2 and later versions.]  
+[Supported in .NET Core 5 and later versions.]  
   
- Configures how the debugger handles in-memory updates to metadata within the target process.  
+ Enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. It is the same information that would be shown running the SOS "eeheap-loader" command.
```
Comment (noahfalk) [inline docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md:21]: Previously I remember seeing that comment about handling in-memory updates in three different spots in the documentation you added. Did we get all the places updated?

Comment (mikelle-rogers) [reply docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md]: I looked in the files that I have changed and it looks like all of them say the correct info.

File: docs/framework/unmanaged-api/debugging/icordebugmodule4-interface.md [hunk 3]
```diff
@@ -50,8 +50,8 @@ interface ICorDebugModule4 : IUnknown
  **Header:** CorDebug.idl, CorDebug.h  
   
  **Library:** CorGuids.lib  
-  
- **.NET Framework Versions:** 4.5, 4, 3.5 SP1
+ 
+ **.NET Core Supported Versions:** 5.0 and above.
```
Comment (hoyosjs) [inline docs/framework/unmanaged-api/debugging/icordebugmodule4-interface.md:54]:
```suggestion
 **.NET Versions:** [!INCLUDE[net_current_v452plus](../../../../includes/net-core-50-md.md)] 
```

File: docs/framework/unmanaged-api/debugging/icordebugprocess11-interface.md [hunk 4]
```diff
@@ -33,8 +33,8 @@ topic_type:
  **Header:** CorDebug.idl, CorDebug.h  
   
  **Library:** CorGuids.lib  
-  
- **.NET Framework Versions:** [!INCLUDE[net_current_v452plus](../../../../includes/net-current-v452plus-md.md)]  
+
+ **.NET Core Supported Versions:** 5.0 and above.
```
Comment (hoyosjs) [inline docs/framework/unmanaged-api/debugging/icordebugprocess11-interface.md:37]:
```suggestion
 **.NET Versions:** [!INCLUDE[net_current_v452plus](../../../../includes/net-core-50-md.md)]
```

File: docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md [hunk 5]
```diff
@@ -42,7 +42,7 @@ HRESULT EnumerateLoaderHeapMemoryRegions(
 
  **Library:** CorGuids.lib  
 
- **.NET Framework Versions:** [!INCLUDE[net_current_v452plus](../../../../includes/net-current-v452plus-md.md)]  
+ **.NET Core Versions:** 5.0 and later
```
Comment (hoyosjs) [inline docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md:45]:
```suggestion
 **.NET Versions:** [!INCLUDE[net_current_v452plus](../../../../includes/net-core-50-md.md)]
```

File: docs/framework/unmanaged-api/debugging/icordebugmodule4-interface.md [hunk 6]
```diff
@@ -50,8 +50,8 @@ interface ICorDebugModule4 : IUnknown
  **Header:** CorDebug.idl, CorDebug.h  
   
  **Library:** CorGuids.lib  
-  
- **.NET Framework Versions:** 4.5, 4, 3.5 SP1
+
+ **.NET Versions:** [!INCLUDE[net_current_v452plus](../../../../includes/net-core-50-md.md)]
```
Comment (hoyosjs) [inline docs/framework/unmanaged-api/debugging/icordebugmodule4-interface.md:54]:
```suggestion
 **.NET Versions:** [!INCLUDE[net-core-50-plus](../../../../includes/net-core-50-md.md)]  
```

File: docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md [hunk 7]
```diff
@@ -42,7 +42,7 @@ HRESULT EnumerateLoaderHeapMemoryRegions(
 
  **Library:** CorGuids.lib  
 
- **.NET Framework Versions:** [!INCLUDE[net_current_v452plus](../../../../includes/net-current-v452plus-md.md)]  
+ **.NET Versions:** [!INCLUDE[net_current_v452plus](../../../../includes/net-core-50-md.md)]
```
Comment (hoyosjs) [inline docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md:45]:
```suggestion
 **.NET Versions:** [!INCLUDE[net-core-50-plus](../../../../includes/net-core-50-md.md)]  
```

File: docs/framework/unmanaged-api/debugging/icordebugprocess11-interface.md [hunk 8]
```diff
@@ -33,8 +33,8 @@ topic_type:
  **Header:** CorDebug.idl, CorDebug.h  
   
  **Library:** CorGuids.lib  
-  
- **.NET Framework Versions:** [!INCLUDE[net_current_v452plus](../../../../includes/net-current-v452plus-md.md)]  
+
+ **.NET Versions:** [!INCLUDE[net_current_v452plus](../../../../includes/net-core-50-md.md)]
```
Comment (hoyosjs) [inline docs/framework/unmanaged-api/debugging/icordebugprocess11-interface.md:37]:
```suggestion
 **.NET Versions:** [!INCLUDE[net-core-50-plus](../../../../includes/net-core-50-md.md)]  
```

**Review by hoyosjs — APPROVED** [review]

File: docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md (hunk 2 above)
Comment (mikelle-rogers) [reply docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md]: I believe so, I searched through the files I have changed

**Review by hoyosjs — APPROVED** [review]

File: docs/framework/unmanaged-api/debugging/debugging-interfaces.md [hunk 9]
```diff
@@ -318,7 +318,7 @@ This section describes the unmanaged interfaces that handle the debugging of a p
  Logically extends the [ICorDebugProcess](icordebugprocess-interface.md) interface to enable or disable certain types of [ICorDebugManagedCallback2](icordebugmanagedcallback2-interface.md) exception callbacks.  
 
  [ICorDebugProcess11 Interface](icordebugprocess11-interface.md)\
- Provides a method that configures the debugger to handle in-memory metadata updates in the target process.  
+ Provides a method that enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. It is the same information that would be shown running the SOS "eeheap -loader" command.
```
Comment (gewarren) [inline docs/framework/unmanaged-api/debugging/debugging-interfaces.md:321]:
```suggestion
 Provides a method that enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. The information returned is the same that would be shown by using the SOS `eeheap -loader` command.  
```

File: docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md [hunk 10]
```diff
@@ -16,9 +16,9 @@ topic_type:
 ---
 # ICorDebugProcess11::EnumerateLoaderHeapMemoryRegions Method
 
-[Supported in .NET Framework 4.5.2 and later versions.]  
+[Supported in .NET Core 5 and later versions.]
```
Comment (gewarren) [inline docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md:19]:
```suggestion
[Supported in .NET 5 and later versions.]  
```

File: docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md [hunk 11]
```diff
@@ -16,9 +16,9 @@ topic_type:
 ---
 # ICorDebugProcess11::EnumerateLoaderHeapMemoryRegions Method
 
-[Supported in .NET Framework 4.5.2 and later versions.]  
+[Supported in .NET Core 5 and later versions.]  
   
- Configures how the debugger handles in-memory updates to metadata within the target process.  
+ Enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. It is the same information that would be shown running the SOS "eeheap-loader" command.
```
Comment (gewarren) [inline docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md:21]:
```suggestion
 Enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. The information returned is the same that would be shown by using the SOS `eeheap-loader` command.
```

File: docs/framework/unmanaged-api/debugging/icordebugprocess11-interface.md [hunk 12]
```diff
@@ -14,15 +14,15 @@ topic_type:
 ---
 # ICorDebugProcess11 Interface
 
-[Supported in .NET Framework 4.5.2 and later versions.]  
-  
- Provides a method that configures the debugger to handle in-memory metadata updates in the target process.  
+[Supported in .NET Core 5.0 and later versions.]
```
Comment (gewarren) [inline docs/framework/unmanaged-api/debugging/icordebugprocess11-interface.md:17]:
```suggestion
[Supported in .NET 5 and later versions.]  
```

File: docs/framework/unmanaged-api/debugging/icordebugprocess11-interface.md [hunk 13]
```diff
@@ -14,15 +14,15 @@ topic_type:
 ---
 # ICorDebugProcess11 Interface
 
-[Supported in .NET Framework 4.5.2 and later versions.]  
-  
- Provides a method that configures the debugger to handle in-memory metadata updates in the target process.  
+[Supported in .NET Core 5.0 and later versions.]  
   
+Provides a method that enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. It is the same information that would be shown running the SOS "eeheap -loader" command.
```
Comment (gewarren) [inline docs/framework/unmanaged-api/debugging/icordebugprocess11-interface.md:19]:
```suggestion
Provides a method that enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. The returned information is the same that would be shown by using the SOS `eeheap -loader` command.
```

File: docs/framework/unmanaged-api/debugging/icordebugprocess11-interface.md [hunk 14]
```diff
@@ -14,15 +14,15 @@ topic_type:
...
+
 ## Methods  
   
 |Method|Description|  
 |------------|-----------------|  
-|[EnumerateLoaderHeapMemoryRegions Method](icordebugprocess11-enumerateloaderheapmemoryregions-method.md)|Sets a value that determines how the debugger handles in-memory updates to metadata within the target process.|  
+|[EnumerateLoaderHeapMemoryRegions Method](icordebugprocess11-enumerateloaderheapmemoryregions-method.md)|enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. It is the same information that would be shown running the SOS "eeheap -loader" command.|
```
Comment (gewarren) [inline docs/framework/unmanaged-api/debugging/icordebugprocess11-interface.md:25]:
```suggestion
|[EnumerateLoaderHeapMemoryRegions Method](icordebugprocess11-enumerateloaderheapmemoryregions-method.md)| Enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. The information returned is the same that would be shown by using the SOS `eeheap -loader` command. |
```

**Review by gewarren — COMMENTED** [review]
Shoot, I missed the merge. Do you want me to make these suggested changes or can you?

**Comment by hoyosjs** [comment]
@mikelle-rogers do you have bandwidth to take on these changes

**Comment by mikelle-rogers** [comment]
> Shoot, I missed the merge. Do you want me to make these suggested changes or can you?

I can make the changes!

**Pull request merged**